apps/prairielearn/python/traverse.py

```diff
@@ -28,6 +28,8 @@
     }
 )
 
+UNESCAPED_ELEMENTS = frozenset({"script"})
```

Comment:
Same issue happens with `<style>` elements (see pl-matching element):

```html
<style>
#pl-matching-ed8ae1d3-677e-472d-a270-60b25c7b1361 .pl-matching-options ol &gt; li:before {
    content: counter(item, lower-alpha) &quot;.&quot;;
}
</style>
```

Reply:
Makes sense. I'll go ahead and add, but I still think that there might be a cleaner way of handling these edge cases than what's currently being done.

Reply:
Ahh, now I remember why we did this. It's because escape sequences were getting translated into the characters they represented by the library. Hopefully adding this exception doesn't break anything, because I think it's now impossible to have something like `&lt;` in a script tag.